test/plugins/features/movies/index.test.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
+
+const Movies = require('../../../../lib/server');
+
+describe('movies integration', () => {
+
+  describe('create', () => {
+
+    it('creates a movie', () => {
+      return Movies.inject({
```

Comment:
if you wanna be fancy, you can go ahead and use `async` and `await` for the integration test or for future code changes

Reply:
haven't used async/await too much in the past but this is good practice. hopefully i did it right!